mavros/src/lib/enum_to_string.cpp

```diff
@@ -286,26 +287,26 @@ std::string to_string(ADSB_ALTITUDE_TYPE e)
 // ]]]
 //! ADSB_EMITTER_TYPE values
 static const std::array<const std::string, 20> adsb_emitter_type_strings{{
-/*  0 */ "NO_INFO",                       // 
-/*  1 */ "LIGHT",                         // 
-/*  2 */ "SMALL",                         // 
-/*  3 */ "LARGE",                         // 
-/*  4 */ "HIGH_VORTEX_LARGE",             // 
-/*  5 */ "HEAVY",                         // 
-/*  6 */ "HIGHLY_MANUV",                  // 
-/*  7 */ "ROTOCRAFT",                     // 
-/*  8 */ "UNASSIGNED",                    // 
-/*  9 */ "GLIDER",                        // 
-/* 10 */ "LIGHTER_AIR",                   // 
-/* 11 */ "PARACHUTE",                     // 
-/* 12 */ "ULTRA_LIGHT",                   // 
-/* 13 */ "UNASSIGNED2",                   // 
-/* 14 */ "UAV",                           // 
-/* 15 */ "SPACE",                         // 
-/* 16 */ "UNASSGINED3",                   // 
-/* 17 */ "EMERGENCY_SURFACE",             // 
-/* 18 */ "SERVICE_SURFACE",               // 
-/* 19 */ "POINT_OBSTACLE",                // 
+/*  0 */ "NO_INFO",                       //
+/*  1 */ "LIGHT",                         //
+/*  2 */ "SMALL",                         //
+/*  3 */ "LARGE",                         //
+/*  4 */ "HIGH_VORTEX_LARGE",             //
+/*  5 */ "HEAVY",                         //
+/*  6 */ "HIGHLY_MANUV",                  //
+/*  7 */ "ROTOCRAFT",                     //
+/*  8 */ "UNASSIGNED",                    //
+/*  9 */ "GLIDER",                        //
+/* 10 */ "LIGHTER_AIR",                   //
+/* 11 */ "PARACHUTE",                     //
+/* 12 */ "ULTRA_LIGHT",                   //
+/* 13 */ "UNASSIGNED2",                   //
+/* 14 */ "UAV",                           //
+/* 15 */ "SPACE",                         //
+/* 16 */ "UNASSGINED3",                   //
+/* 17 */ "EMERGENCY_SURFACE",             //
+/* 18 */ "SERVICE_SURFACE",               //
+/* 19 */ "POINT_OBSTACLE",                //
```

Comment:
Should not be done! That brokes cog.py md5sum.
If that done uncrustify, then re-run cog on that file (possible you need `cog.py -x` first).